### PR TITLE
feat(suite): Add cursor: not-allowed to some components when disabled

### LINKED ISSUE #12945: Add cursor:not-allowed to disable components
To all interactive/input components in our design system, we want to add `cursor: not-allowed` when they are disabled.

Components:
```
Input
Button
ButttonGroup (maybe it will work when you change it in Button)
Radio
Checkbox
AutoscalingInput
Range
Select
SelectBar (?)
Switch
Textarea

ButtonGroup
IconButton
PinButton
TextButton
TooltipButton
```

### PR DESCRIPTION
**Pull request by hstastna**
## Description

Add `cursor: not-allowed` for cursor to look nicer to some more of the interactive/input components in our design system, when being disabled.

Additionally, add args and argTypes in storybook for components that are missing them so one could easily change the values of selected component's props in storybook.

This PR takes care about the following components:
- `AutoscalingInput`
- `Range`
- `Select`
- `SelectBar`
- `Switch`

Also for `Button` component, add way to choose from all the available icons in storybook, incl. choosing the option "No icon" in storybook, as the _icon_ prop is optional.

For `AutoscalingInput`, add story for playing with its props, use the existing one to just show the examples.

TODO (in followup PR):
- [ ] `Textarea`
- [ ] `IconButton`
- [ ] `PinButton`
- [ ] `TextButton`
- [ ] `TooltipButton`

## Related Issue

Resolve https://github.com/trezor/trezor-suite/issues/12945

## Screenshots:
**Before:**
Example: `Label` component
label_before

**After:**
Example: `Label` component: updated cursor when disabled + added more controls to storybook
label_after

Button component: now you can easily choose an icon if you want, or also not:
icons

### CODE REVIEW

File: packages/components/src/components/form/SelectBar/SelectBar.tsx [hunk 1]
```diff
@@ -216,7 +217,7 @@ export const SelectBar: <V extends ValueTypes>(props: SelectBarProps<V>) => JSX.
                 {options.map(option => (
                     <Option
                         key={String(option.value)}
-                        onClick={handleOptionClick(option)}
+                        onClick={handleOptionClick(isDisabled, option)}
```
Comment (peter-sanderson) [inline packages/components/src/components/form/SelectBar/SelectBar.tsx]: I think `isDisabled` dont need to be passed as argument, it is already available in the handler from component props.

File: packages/components/src/components/form/Select/Select.tsx [hunk 2]
```diff
@@ -217,6 +217,8 @@ const Wrapper = styled.div<WrapperProps>`
         border: none;
         z-index: ${zIndices.base};
     }
+
+    ${({ $isDisabled }) => $isDisabled && `pointer-events: auto; cursor: not-allowed;`};
```
Comment (peter-sanderson) [inline packages/components/src/components/form/Select/Select.tsx]: This maybe works but I think it should be: 

```ts
    ${({ $isDisabled }) =>
        $isDisabled &&
        css`
            pointer-events: auto;
            cursor: not-allowed;
        `};
```

Or at least we use it this way everywhere else. Also with this 
```
css`...`
```
 auto-format works and it then places every CSS directive on separate line.

Comment (hstastna) [reply packages/components/src/components/form/Select/Select.tsx]: Right! And it's more readable like this. Don't know why I've used that one line format here instead :)

File: packages/components/src/components/form/SelectBar/SelectBar.tsx (hunk 1 above)
Comment (hstastna) [reply packages/components/src/components/form/SelectBar/SelectBar.tsx]: I think I had a good reason for this. But I'll take a look again. Now seems that `isDisabled` really doesn't need to be there. Thanks!

**Review by peter-sanderson — APPROVED** [review]

**Pull request merged**